addons/website_twitter/models/twitter.py

```diff
@@ -41,52 +40,51 @@ def _request(self, website, url, params=None):
                           e.code, e.msg, e.fp.read())
             raise
 
-    def _refresh_favorite_tweets(self, cr, uid, context=None):
+    @api.model
+    def _refresh_favorite_tweets(self):
         ''' called by cron job '''
-        website = self.pool['website']
-        ids = self.pool['website'].search(cr, uid, [('twitter_api_key', '!=', False),
-                                                    ('twitter_api_secret', '!=', False),
-                                                    ('twitter_screen_name', '!=', False)],
-                                          context=context)
-        _logger.debug("Refreshing tweets for website IDs: %r", ids)
-        website.fetch_favorite_tweets(cr, uid, ids, context=context)
+        website = self.env['website']
+        ids = self.env['website'].search([('twitter_api_key', '!=', False),
+                                          ('twitter_api_secret', '!=', False),
+                                          ('twitter_screen_name', '!=', False)])
+        _logger.debug("Refreshing tweets for website IDs: %r", ids.id)
+        website.fetch_favorite_tweets()
 
-    def fetch_favorite_tweets(self, cr, uid, ids, context=None):
-        website_tweets = self.pool['website.twitter.tweet']
+    @api.multi
+    def fetch_favorite_tweets(self):
+        website_tweets = self.env['website.twitter.tweet']
         tweet_ids = []
-        for website in self.browse(cr, uid, ids, context=context):
+        for website in self.search([]):
```

Comment:
That changes the semantics of the method quite a bit, and makes the search in `_refresh_favorite_tweets` completely useless...